### PR TITLE
NOT READY : Online multiplayer integration tests without RPC server dependency.

### PR DESCRIPTION
**Pull request by al-mehedi**

### CODE REVIEW

**Comment by Wandalen** [comment]
@dmvict please review

**Comment by dmvict** [comment]
The PR looks good. 

I've tested it and I've got the fails 
```bash
failures:

---- online_multiplayer_game_tests::online_game_new stdout ----
thread 'online_multiplayer_game_tests::online_game_new' panicked at 'Failed to connect gRPC server', module/cui/./src/main.rs:487:7

---- online_multiplayer_game_tests::online_game_join stdout ----
thread 'online_multiplayer_game_tests::online_game_join' panicked at 'Failed to connect gRPC server', module/cui/./src/main.rs:518:7
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```


Please, fix fails and pull request will be approved and merged.

**Comment by al-mehedi** [comment]
@dmvict fixed the issue.

File: module/multiplayer/proto/chess_game.proto [hunk 1]
```diff
@@ -31,12 +31,6 @@ message Blank
 {
 }
 
-message Player
```
Comment (dmvict) [inline module/multiplayer/proto/chess_game.proto:34]: Explain this change, please. 

I know that this structure is designed for instance of any player, but the structure GamePlayer is designed for instance of player in multiplayer. 
GamePlayer extends player. 
Does it?

File: module/multiplayer/proto/chess_game.proto [hunk 2]
```diff
@@ -63,18 +57,9 @@ message Board
 }
 
 message MultiplayerGame {
-  string id = 1;
+  string game_id = 1;
   repeated GamePlayer players = 2;
-  enum GameStatus {
```
Comment (dmvict) [inline module/multiplayer/proto/chess_game.proto:68]: Please, explain this change. 

What the logic do you implement now? How to get data about game state, exactly forfeit ( `gg` ) ?

**Pull request merged**